modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -245,7 +254,7 @@ public Response getEpisodeAndSeriesById(
               name = "sign", type = RestParameter.Type.BOOLEAN)
   }, reponses = { @RestResponse(description = "The request was processed successfully.", responseCode = HttpServletResponse.SC_OK) }, returnDescription = "The search results, formatted as xml or json.")
   public Response getEpisode(@QueryParam("id") String id, @QueryParam("q") String text,
-          @QueryParam("sid") String seriesId, @QueryParam("sort") String sort, @QueryParam("tag") String[] tags, @QueryParam("flavor") String[] flavors,
+          @QueryParam("sid") String seriesId, @QueryParam("sname") String seriesName, @QueryParam("sort") String sort, @QueryParam("tag") String[] tags, @QueryParam("flavor") String[] flavors,
           @QueryParam("limit") int limit, @QueryParam("offset") int offset, @QueryParam("admin") boolean admin,
```

Comment:
That new sname param should to be added to the REST docs annotation describing the params for this method. It would be helpful to clarify in the REST doc annotation text that the used of "sname" param here assumes a unique series titles. i.e. not something like "Intro to Science 101" duplicated across different terms (like our site does).

Reply:
Agreed, fixed that one.